server/src/test/java/org/apache/cassandra/sidecar/cdc/CDCLogCacheTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.cdc;
+
+import com.google.common.base.Preconditions;
+
+import org.junit.jupiter.api.Test;
+import io.vertx.core.Vertx;
+import org.apache.cassandra.sidecar.ExecutorPoolsHelper;
+import org.apache.cassandra.sidecar.cluster.InstancesConfig;
+import org.apache.cassandra.sidecar.concurrent.ExecutorPools;
+
+import static org.mockito.Mockito.mock;
+
+class CDCLogCacheTest
+{
+    Vertx vertx = Vertx.vertx();
+
+    /**
+     * Failing to clean up shouldn't fail to initialize the class
+     */
+    @Test
+    void testCleanupErrorDoesntPreventInitialization()
+    {
+        new FailingCDCLogCache(ExecutorPoolsHelper.createdSharedTestPool(vertx),
+                mock(InstancesConfig.class));
+    }
```

Comment:
Can you add more test cases? For example, hard link creation and cleanup, cache expiry, etc.

Reply:
Added!